src/libostree/ostree-repo.c

```diff
@@ -257,13 +257,18 @@ _ostree_repo_remote_name_is_file (const char *remote_name)
   return g_str_has_prefix (remote_name, "file://");
 }
 
+/**
+ * ostree_repo_get_remote_option:
+ * @self:
+ *
```

Comment:
I think we need to do more than stub this out, otherwise e.g. introspection won't understand the `char **out_value` and it won't be accessible to scripts.

Something like:

```
@self: Repo
@remote_name: Name
@option_name: Option
@default_value: (allow-none): Value returned if @option_name is not present
@out_value: (out): Return location for value
@error: Error

OSTree remotes are represented by keyfile groups, formatted like:
`[remote "remotename"]`. This function returns a value named @option_name
underneath that group.
```

?